qtrade/questrade.py

```diff
@@ -759,4 +759,8 @@ def __del__(self):
 
     @staticmethod
     def _valid_intervals():
-        return set(["FifteenMinutes", "OneDay", "OneWeek", "OneMonth", "OneYear"])
+        return set(["OneMinute", "TwoMinutes", "ThreeMinutes", "FourMinutes", "FiveMinutes",
```

Comment:
Have you tried all of them on your local branch? IIRC they used to enable much more granular intervals, but they don't work anymore.

Reply:
Oh I see, ya I just looped through pulling `AAPL` info for all intervals, which seems to work on my end:

```
for interval in ["OneMinute", "TwoMinutes", "ThreeMinutes", "FourMinutes", "FiveMinutes",
            "TenMinutes", "FifteenMinutes", "TwentyMinutes", "HalfHour", "OneHour", "TwoHours",
            "FourHours", "OneDay", "OneWeek", "OneMonth", "OneYear"]:
    apple = q.get_historical_data('AAPL', '2025-04-01', '2025-04-15', interval)
    print([i['close'] for i in apple][0:10])
```

Reply:
The caveat is that for a granularity finer than a day, data from Questrade seems to be **limited to the past 3 months**. No error is thrown if an earlier date is specified, the request just pulls everything it has.